apps/meteor/client/sidebar/header/EditStatusModal.tsx

```diff
@@ -31,6 +33,9 @@ const EditStatusModal = ({ onClose, userStatus, userStatusText }: EditStatusModa
 		if (statusText && statusText.length > USER_STATUS_TEXT_MAX_LENGTH) {
 			return setStatusTextError(t('Max_length_is', USER_STATUS_TEXT_MAX_LENGTH));
 		}
+		if (statusText && statusText.length < USER_STATUS_TEXT_MAX_LENGTH) {
+			setCustomStatus(e.currentTarget.value);
```

Comment:
This code will never run, because there's an exactly equal if statement above that returns the fuction.

Also, it is preferable that you save the custom status only after it is successfully saved. You can move this to the `handleSaveStatus` callback below, inside the `trycatch` block (after the `setUserStatus` function call)

Reply:
Thank you for the review and guidance :) I've made the changes now! 

PS. The if statement was there to handle edge cases (status is longer than 120 chars), it was the exact opposite condition than the one above. In hindsight, I should've probably used an else statement.